src/validations/admins.js

```diff
@@ -1,5 +1,40 @@
 const Joi = require('joi');
 
+const validateUpdate = (req, res, next) => {
+  const adminValidation = Joi.object({
+    firstName: Joi.string()
+      .min(3)
+      .max(15),
+    lastName: Joi.string()
+      .min(3)
+      .max(15),
+    dni: Joi.number()
+      .positive()
+      .min(10000000)
+      .max(99999999),
+    phone: Joi.number()
```

Comment:
phone must be a string
changing this, min and max will be the length of the string (9 and 12)